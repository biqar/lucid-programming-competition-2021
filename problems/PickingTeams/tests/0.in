6
+ Alice Bob
+ Charlie David
? Alice Charlie
? Bob Alice
+ Bob Charlie
? Alice David
