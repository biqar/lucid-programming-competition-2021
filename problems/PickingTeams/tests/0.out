no
yes
yes
1
